fitnessllm_dataplatform/services/api_interface.py

```
@@ -4,6 +4,8 @@
 class APIInterface:
     """API Interface for FitnessLLM Data Platform."""
 
+    uid: str
+
     def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)